apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx

```diff
@@ -429,6 +429,9 @@ export const SignedVerdictOverview: React.FC = () => {
             created: new Date().toString(),
             modified: new Date().toString(),
           },
+          isHeightenedSecurityLevel: workingCase.isHeightenedSecurityLevel
```

Comment:
What is the reasoning here? Why can't this be `isHeightenedSecurityLevel: false`? Aren't we always turning the heightened security level off when we share requests?

Reply:
Most of the time this field is undefined and I don't want to set it to false if it is undefined.